python/taichi/lang/matrix.py

```diff
@@ -146,9 +146,6 @@ def is_global(self):
                 "Matrices with mixed global/local entries are not allowed"
         return results[0]
 
-    def is_pyconstant(self):
-        return all([not isinstance(e, expr.Expr) for e in self.entries])
```

Comment:
This line returns True for global vectors initialized as `ti.Vector([0.0,-0.005,0.0])`, which leads to this bug.